### PR TITLE
[Doc Improvement][Team relationship expandable update][3203414]

### PR DESCRIPTION
**Pull request by v-sdhakshina**

### CODE REVIEW

**Review by Nivedipa-MSFT — APPROVED** [review]

File: api-reference/v1.0/resources/team.md [hunk 1]
```diff
@@ -84,19 +84,19 @@ For a POST request example, see [Request (create team in migration state)](/micr
 
 ## Relationships
 
-| Relationship | Type | Description |
+| Relationship | Type | Description | Expandable |
 |:---------------|:--------|:----------|
-|allChannels|[channel](channel.md) collection|List of channels either hosted in or shared with the team (incoming channels).|
```
Comment (Wajeed-msft) [inline api-reference/v1.0/resources/team.md:89]: @surbhigupta12  - same page in beta doesn't have these changes. Also, this PR makes table with additional parameter which is not present in any other page available. 

Ex: https://learn.microsoft.com/en-us/graph/api/resources/channel?view=graph-rest-beta#relationships

Please confirm this with Graph team before merging the PR.

**Review by Wajeed-msft — APPROVED** [review]

File: api-reference/v1.0/resources/team.md (hunk 1 above)
Comment (v-sdhakshina) [reply api-reference/v1.0/resources/team.md:89]: Updated the Beta doc as suggested

File: api-reference/beta/resources/team.md [hunk 2]
```diff
@@ -87,22 +87,22 @@ For a POST request example, see [Request (create team in migration state)](/micr
...
+|:---------------|:--------|:----------|:----------|
+|allChannels|[channel](channel.md) collection|List of channels either hosted in or shared with the team (incoming channels).| No |
+|channels|[channel](channel.md) collection|The collection of channels and messages associated with the team.| No |
+|incomingChannels|[channel](channel.md) collection|List of [channels](../resources/channel.md) shared with the team.| No |
+|installedApps|[teamsAppInstallation](teamsappinstallation.md) collection|The apps installed in this team.| No |
+|members|[conversationMember](../resources/conversationmember.md) collection|Members and owners of the team.| No |
+|owners|[user](user.md)| The list of this team's owners. Currently, when creating a team using application permissions, exactly one owner must be specified. When using user delegated permissions, no owner can be specified (the current user is the owner). Owner must be specified as an object ID (GUID), not a UPN. | No |
+|operations|[teamsAsyncOperation](teamsasyncoperation.md) collection| The async operations that ran or are running on this team. | No |
+|photo|[profilePhoto](../resources/profilephoto.md)|The team photo.| Yes |
```
Comment (AkJo) [inline api-reference/beta/resources/team.md:99]: I am not sure if this is expandable.

File: api-reference/v1.0/resources/team.md [hunk 3]
```diff
@@ -84,19 +84,19 @@ For a POST request example, see [Request (create team in migration state)](/micr
...
+|allChannels|[channel](channel.md) collection|List of channels either hosted in or shared with the team (incoming channels).| No |
+|channels|[channel](channel.md) collection|The collection of channels and messages associated with the team.| No |
+|incomingChannels|[channel](channel.md) collection|List of [channels](../resources/channel.md) shared with the team.| No |
+|installedApps|[teamsAppInstallation](teamsappinstallation.md) collection|The apps installed in this team.| No |
+|members|[conversationMember](../resources/conversationmember.md) collection|Members and owners of the team.| No |
+|operations|[teamsAsyncOperation](teamsasyncoperation.md) collection| The async operations that ran or are running on this team. | No |
+|photo|[profilePhoto](../resources/profilephoto.md)| The profile photo for the team. | No |
+|[primaryChannel](../api/team-get-primarychannel.md)|[channel](channel.md)| The general channel for the team. | Yes |
+|schedule|[schedule](schedule.md)| The schedule of shifts for this team.| No |
+|tags|[teamworkTag](teamworktag.md) collection|The tags associated with the team.| No |
```
Comment (AkJo) [inline api-reference/v1.0/resources/team.md:98]: I do not think that this is the right way to represent this data.

Whether a relationship is expandable or not is very contextualized to the exact API. The same relationship might be non-expandable in GET /me/joinedTeams, but might be expandable in GET /teams/id/

Comment (AkJo) [reply api-reference/v1.0/resources/team.md:98]: We should not commit any of these changes.

**Comment by v-sdhakshina** [comment]
Closing this PR as there is no doc update required for this git issue.

**Pull request closed without merging**